source/_page_templates/ability_template.rst

```diff
@@ -0,0 +1,58 @@
+.. [insert ability name here, lowercase, one word]:
+############
+ABILITY NAME
+############
+
+Description
+###########
+
+Paste description here. Insert gif here |ABILNAMEHERE1|
+
+
+Usage
+#####
+Below are the steps to use this ability
+
+Insert in-game gif here (add another substition)
```

Comment:
How does this differ from the above gif?

Reply:
Its the usage gif, slowed down showing the specific steps

Reply:
Might be worth saying that then.